configs/next/packages/mpfr/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="GNU MPFR Library"
 ATSRC_PACKAGE_VER=3.1.7
```

Comment:
I'm not opposed to this automatic update, but I think we should update `ATSRC_PACKAGE_VER` to `4.1.0-dev`, because that's the version reported by:

```
$ svn cat https://scm.gforge.inria.fr/anonscm/svn/mpfr/trunk/VERSION
4.1.0-dev
```

Reply:
> I'm not opposed to this automatic update

This means that I believe this patch should be committed with or without the change to `ATSRC_PACKAGE_VER `.  If changing the commit is possible from within the pull request, I suggest this be done.